R/opts.R

```diff
@@ -248,6 +248,10 @@ gp_opts <- function(basis_prop = 0.2,
 #'  to map latent infections (convolved to date of report). If none empty a mean
 #'  (`mean`) and standard deviation (`sd`) needs to be supplied defining the normally
 #'  distributed scaling factor.
+#' @param likelihood Logical, defaults to `TRUE`. Should the likelihood be
+#'  included in the model
+#' @param return_likelihood Logical, defaults to `FALSE`. Should the likelihood
+#'  be returned by the model.
```

Comment:
Couldn't we remove `return_likelihood` and add `"likelihood"` to the list of supported outputs?
By using the `output` argument like:
```r
    output = c(
      "fit",
      "latest",
      "likelihood",
      "plots",
      "regions",
      "samples",
      "summary",
      "timing"
    )
```

Reply:
Also, the `create_obs_model` test fails due to the changed number of arguments:
https://github.com/epiforecasts/EpiNow2/blob/a582f210bb8a4b487aff1722f99777119fa83c3b/tests/testthat/test-create_obs_model.R#L6

Reply:
Thanks @hsbadr 

Can't add to output as that is only supported in `epinow` and not in the internal functions. That being said I am kicking off a round of changes with the first focus being some work on the stan model. Once that is done going to look at improving the interface where this might come in (and probably some fairly big other changes) - just kicking the tires with a couple of PRs to get back up to speed on the code base.

Reply:
Sounds great. Thanks for heads up!